src/main/java/saps/scheduler/core/SchedulerMain.java

```diff
@@ -10,6 +11,18 @@
 	private static final Logger LOGGER = Logger.getLogger(SchedulerMain.class);
 
 	public static void main(String[] args) throws Exception {
+		String confPath = args[0];
```

Comment:
another option for this would be to keep the conf files always in the same place, which is the best approach?

Reply:
Receive the path of the configuration file as a parameter. I believe it is advantageous, I can switch between configuration files and directories just by changing the path.